README.md

```diff
@@ -1,13 +1,29 @@
 # Article Linking
 ![Python application](https://github.com/georgetown-cset/article-linking/workflows/Python%20application/badge.svg)
 
-This repository contains a description and supporting code for CSET's current method of
-cross-dataset article linking. Note that we use "article" very loosely, although in a way that to our knowledge
-is fairly consistent across corpora. Books, for example, are included.
+At CSET, we aim to produce a more comprehensive set of scholarly literature by ingesting multiple sources and then
+deduplicating articles. This repository contains CSET's current method of cross-dataset article linking. Note that we
+use "article" very loosely, although in a way that to our knowledge is fairly consistent across corpora. Books, for
```

Comment:
Is "consistent across corpora" meant to say that "articles" are all referring to the same type of thing, regardless of the source dataset? Or meant to say that our definition of "article" matches similar definitions across other aggregated pubs datasets?

Reply:
Yeah, the first one - I'll make that clearer